pkg/agent/datamodel/sig_config.go

```diff
@@ -542,6 +545,13 @@ var (
 		Version:       LinuxSIGImageVersion,
 	}
 
+	SIGUbuntuContainerd2404CVMGen2ImageConfigTemplate = SigImageConfigTemplate{
+		ResourceGroup: AKSUbuntuResourceGroup,
+		Gallery:       AKSUbuntuGalleryName,
+		Definition:    "2404gen2CVMcontainerd",
+		Version:       LinuxSIGImageVersion,
```

Comment:
since you add 2404+CVM distro info, better to enable 'build2404cvmgen2containerd' by default.